### PR TITLE
added photos component

### PR DESCRIPTION
**Pull request by TylerNguyen21**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -1,12 +1,43 @@
 import React from 'react';
+import Photos from './Photos.jsx';
 
-class App extends React.Component {
+class Gallery extends React.Component {
     constructor() {
         super()
```
Comment (ngodavidhuy) [inline client/src/components/App.jsx:6]: It's a good practice to always includes props inside of constructor and props

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
+    grabPhotos () {
+        fetch(`/${this.state.resturantId}/photos`, {
+            method: 'GET',
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        })
+        .then(res => res.json())
+        .then((gallery) => {
+            console.log(gallery);
```
Comment (ngodavidhuy) [inline client/src/components/App.jsx:27]: You sure you want to leave this console.log here?

File: server/server.js [hunk 3]
```diff
@@ -14,14 +14,13 @@ app.get('/:restId/photos', (req,res) => {
         if (err) {
             throw err;
         } else {
-            res.status(200).json(result);
+            res.status(200);
+            console.log('hello');
```
Comment (ngodavidhuy) [inline server/server.js:18]: Again, the console.log? Do you want to leave it?

**Comment by ngodavidhuy** [comment]
Aside from the remarks I left, it looks good to me!

File: server/server.js (hunk 3 above)
Comment (TylerNguyen21) [reply server/server.js:18]: forgot I left them there thanks for catching them! will fix them before i pull completely!

**Pull request merged**